### PR TITLE
reconsolidate eslint stylelint commitizion etc

### PR DESCRIPTION
**Pull request by McDaddy**
## What type of PR is this?

- [ ] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [x] Chore

## What this PR does / why we need it:
1. introduce f2elint
2. auto lint fix for core
3. remove eslint/stylelint/commitizen config to monorepo root
4. add prettier
5. add spell check config

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/tsconfig.json [hunk 1]
```diff
@@ -1,14 +1,11 @@
 {
   "compilerOptions": {
-    "composite": true,
+    "composite": false,
```
Comment (daskyrk) [inline shell/tsconfig.json:3]: why set to false

Comment (McDaddy) [reply shell/tsconfig.json:3]: image

Comment (McDaddy) [reply shell/tsconfig.json:3]: image

Comment (McDaddy) [reply shell/tsconfig.json:3]: in tsconfig.json of shell, we just need to include source file of shell.  In monorepo, shell refer code like `core/main` which is under tsconfig root, and not `include` either. So set composite to false then ts will find related code automatically

**Review by daskyrk — APPROVED** [review]

**Pull request merged**